In Solaris 11.4, we have normalized man pages. See PSARC 2015/428.
This means that what was section 1m, is now section 8 and what was
section 4 is now section 5. But currently only for Solaris 11.4.

This patch should eventually be sent upstream, but for that to happen
it will need to be reworked into a form that will be acceptable for
all versions of Solaris. Until then, we are just using this Solaris 11.4
specific patch.

--- sudo-1.9.12p2/configure.ac.org
+++ sudo-1.9.12p2/configure.ac
@@ -1805,9 +1805,9 @@
 		if test "$with_AFS" = "yes"; then
 		    AFS_LIBS="-lc -lucb"
 		fi
-		: ${mansectsu='1m'}
-		: ${mansectform='4'}
-		: ${mansectmisc='5'}
+		: ${mansectsu='8'}
+		: ${mansectform='5'}
+		: ${mansectmisc='7'}
 		test -z "$with_pam" && AUTH_EXCL_DEF="PAM"
 		AC_CHECK_FUNCS([priv_set], [PSMAN=1])
 		;;
